lib/mix/tasks/nerves/deps.get.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Mix.Tasks.Nerves.Deps.Get do
+  use Mix.Task
+
+  import Mix.Nerves.IO
+  
+  def run(_argv) do
+    debug_info("Deps.Get Start")
```

Comment:
Shouldn't the debug_info be `Nerves.Deps.Get` start`?

Reply:
Yup. Updated.